src/Microsoft.Framework.PackageManager/Packing/PackManager.cs

```diff
@@ -117,27 +117,24 @@ public bool Package()
                 {
                     runtimeProbePaths = new List<string>();
                     runtimeProbePaths.Add(runtime);
-                    var kreHome = Environment.GetEnvironmentVariable("KRE_HOME");
-                    if (string.IsNullOrEmpty(kreHome))
+                    // TODO: remove KRE_ env var
+                    var dotnetHome = Environment.GetEnvironmentVariable("DOTNET_HOME") ?? Environment.GetEnvironmentVariable("KRE_HOME");
+                    if (string.IsNullOrEmpty(dotnetHome))
                     {
+                        var dotnetGlobalPath = Environment.GetEnvironmentVariable("DOTNET_GLOBAL_PATH");
```

Comment:
FYI, Right now the only place this environment variable is set is in the `kvm` tests. It's an override, but if it isn't specified, it needs to have a default value.

If `DOTNET_GLOBAL_PATH` isn't set, this code should fall back to `%ProgramFiles%\Microsoft .NET Cross-Platform Runtime` (or whatever we finalize on... maybe put it in a constant to make it easier to move around :))